plugins/qeta-react/src/utils/utils.ts

```diff
@@ -49,7 +49,7 @@ export const formatEntityName = (username?: string) => {
 
 export const getEntityTitle = (entity: Entity): string => {
   const stringified = stringifyEntityRef(entity);
-  return formatEntityName(entity.metadata.title ?? stringified) ?? stringified;
+  return entity.metadata.title ?? formatEntityName(stringified) ?? stringified;
```

Comment:
I think we should change this whole function to use `entityPresentationApi`. But that is a bigger refactor I am afraid so let's go with this for now.